packages/datastore/src/listfield.ts

```diff
@@ -168,6 +168,36 @@ class ListField<T extends ReadonlyJSONValue> extends Field<ListField.Value<T>, L
     return { value: clone, change };
   }
 
+  /**
+   * Encode a system patch so that it can be sent across a network.
+   *
+   * @param patch - The patch to encode.
+   *
+   * @returns a JSON value that can be sent across the network.
+   */
+  encodePatch(patch: ListField.Patch<T>): ListField.Patch<T> {
+    return patch.map(part => ({
+      ...part,
+      removedIds: part.removedIds.map(id => encodeId(id)),
+      inserteIds: part.insertedIds.map(id => encodeId(id))
```

Comment:
inserted*

Reply:
Hah, this might explain why it's not quite working yet!